cekit/generator/base.py

```diff
@@ -91,6 +92,11 @@ def add_build_labels(self):
         # was used to build the image
         image_labels.append(Label({'name': 'io.cekit.version', 'value': cekit_version}))
 
+        for label in image_labels:
+            if len(label.value) > 128:
+                # break the first line
```

Comment:
This comment is not entirely true, since it's breaking the line multiple times, if the line is multiple times longer than 128 characters.

Reply:
wdyt of `breaks the line each time it reaches 128 characters`?